sdk/src/main/java/software/amazon/awssdk/iot/V2DeserializationFailureEvent.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ *
+ * This file is generated.
+ */
+
+package software.amazon.awssdk.iot;
+
+/**
+ * An event emitted by a streaming operation when an incoming messages fails to deserialize
+ */
+public class V2DeserializationFailureEvent {
+    private Throwable cause;
+    private byte[] payload;
+    // when topic is available, add it
```

Comment:
Trivial: Should this comment be labeled as a TODO? Or removed?

Reply:
Topic is available in the CRT event data; I'll just add it now.